packages/circuit-ui/tsconfig.json

```diff
@@ -17,6 +17,7 @@
     "noUnusedParameters": true,
     "resolveJsonModule": true,
     "strict": true,
+    "skipLibCheck": true,
```

Comment:
`nanostores` has a type error in its source code. Since this isn't something we can fix ourselves, I've enabled `skipLibCheck` to turn off type checking of dependencies. Our usage of this dependencies is still type-checked.